pollination/imageless_annual_glare/entry.py

```diff
@@ -223,37 +224,35 @@ def annual_imageless_glare(
         sensor_count='{{item.count}}',
         sky_matrix=create_total_sky._outputs.sky_matrix,
         sky_dome=create_sky_dome._outputs.sky_dome,
-        bsdfs=create_rad_folder._outputs.bsdf_folder,
-        schedule=schedule,
-        glare_limit=glare_threshold
+        bsdfs=create_rad_folder._outputs.bsdf_folder
     ):
         pass
 
     @task(
         template=MergeFolderData,
         needs=[annual_imageless_glare]
     )
-    def restructure_glare_autonomy_results(
-        self, input_folder='initial_results/ga', extension='ga'
+    def restructure_daylight_glare_probability_results(
+        self, input_folder='initial_results/dgp', extension='dgp'
     ):
         return [
             {
                 'from': MergeFolderData()._outputs.output_folder,
-                'to': 'metrics/ga'
+                'to': 'results'
             }
         ]
 
     @task(
-        template=MergeFolderData,
-        needs=[annual_imageless_glare]
+        template=AnnualGlareAutonomy,
+        needs=[restructure_daylight_glare_probability_results]
     )
-    def restructure_daylight_glare_probability_results(
-        self, input_folder='initial_results/dgp', extension='dgp'
+    def daylight_glare_autonomy(
+        self, folder='results', schedule=schedule, glare_threshold=glare_threshold
```

Comment:
This will technically work with the luigi recipe but, to get it to work well with queenbee-argo, I think it's better practice to say:
```
folder=restructure_daylight_glare_probability_results._outputs.output_folder
```